### PR TITLE
LPS-152670

### PR DESCRIPTION
**Pull request by alinedoleron**
Related Issue: https://issues.liferay.com/browse/LPS-152670

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please fix the source formatter issues and resend? Thanks

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please fix the source formatter issues and resend? Thanks

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please fix the source formatter issues and resend? Thanks

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please fix the source formatter issues and resend? Thanks

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please fix the source formatter issues and resend? Thanks

**Comment by alinedoleron** [comment]
ci:test:sf

File: modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js [hunk 1]
```diff
@@ -431,6 +434,7 @@ ActionsDropdownRenderer.propTypes = {
 			onClick: PropTypes.string,
 			target: PropTypes.oneOf([
 				'async',
+				'event',
```
Comment (alinedoleron) [inline modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js:437]: Hello, @markocikos. Please, could you review these changes?

File: modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js [hunk 2]
```diff
@@ -111,6 +111,9 @@ export function handleAction(
 
 		window.open(url);
 	}
+	else if (target === 'event') {
+		Liferay.fire(url, {itemId});
+	}
```
Comment (alinedoleron) [inline modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js:116]: Hello, @markocikos. Please, could you review these changes?

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please check if these failures are related to your changes? Thanks

**Comment by liferay-forms** [comment]
Hey @alinedoleron, can you please check if these failures are related to your changes? Thanks

File: modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js (hunk 1 above)
Comment (markocikos) [reply modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/data_renderers/ActionsDropdownRenderer.js:437]: @alinedoleron Please do not add new action `target`s. Our intent is to minimize and hopefully remove the whole API one day. This pattern is providing solutions tailored to individual customizations, which has no place in a general purpose component. If we continue doing this, we will end up with hundreds of targets.

Instead, please use `propsTransformer` API to add your custom action. See [docs](https://liferay.atlassian.net/wiki/spaces/ENGFRONTENDINFRA/pages/2012217428/Frontend+Data+Set+Actions#Custom-Actions) and [a sample](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/resources/META-INF/resources/js/SampleFDSPropsTransformer.js).

**Pull request closed without merging**